source/includes/commit-process.rst

```diff
@@ -0,0 +1,138 @@
+.. step:: Verify the status of mongosync.
+
+   Call the :ref:`progress <c2c-api-progress>` endpoint to determine
+   the status of ``mongosync`` before starting the cutover process.
+   Ensure that the ``mongosync`` process status indicates the
+   following values:
+
+   - ``canCommit`` is ``true``.
+
+   - ``lagTimeSeconds`` is small (near ``0``).
+
+      If ``lagTimeSeconds`` isn't close to ``0`` when the cutover
+      starts, cutover might take a long time.
+
+   The following example returns the status of the synchronization process.
+
+   Request
+   ~~~~~~~
+
+   .. literalinclude:: /includes/api/requests/progress.sh
+      :language: shell
+
+   Response
+   ~~~~~~~~
+
+   .. literalinclude:: /includes/api/responses/progress.json
+      :language: json
+      :copyable: false
+      :emphasize-lines: 5, 8
+
+.. step:: Stop any write operations to the synced collections on the source.
+
+   - If you start ``mongosync`` with ``enableUserWriteBlocking``
+      set to ``true``, ``mongosync`` blocks all write operations on
+      the entire source cluster during the commit (step 4) for you.
+   - If you didn't start ``mongosync`` with
+      ``enableUserWriteBlocking``, ensure that you disable writes.
+      For example, run the :dbcommand:`setUserWriteBlockMode` command on the
+      source cluster:
+
+      .. code-block:: javascript
+
+         db.adminCommand( {
+            setUserWriteBlockMode: 1,
+            global: true 
+         } )
+
+   - If ``mongosync`` uses filtered sync, it's not necessary to
+      disable writes to the entire source cluster. However, you must ensure
+      that you stop write operations for the collections that the
+      filter includes.
```

Comment:
Can we remove this bullet since filtered sync isn't supported with reverse mode? CC @ajayvijayakumar123

Reply:
Fair point. I initially made the suggestion to consolidate the common steps between the reverse and commit pages. If we want to only include the note about filtered sync on the commit page, I think we'd need to do a bit of tweaking. I imagine we can remove this note out of the common commit-process page and only add it for the commit process page.

Alternatively, we could just make the reverse sync tutorial be something like:
1. follow the steps from the cutover process [tutorial](https://deploy-preview-481--docs-cluster-to-cluster-sync.netlify.app/reference/cutover-process/).
2. Call the reverse endpoint to reverse the direction of your sync operation ... mongosync replicates all writes that you performed on your new source cluster after you unblocked writes to the new destination cluster.

That way we would just have the cutover steps on the cutover tutorial. I think I'm in favor of this option now. (Apologies for the back and forth). @ltran-mdb2 @mvankeulen94 thoughts?

Reply:
> Alternatively, we could just make the reverse sync tutorial be something like:
> 
> 1. follow the steps from the cutover process [tutorial](https://deploy-preview-481--docs-cluster-to-cluster-sync.netlify.app/reference/cutover-process/).
> 2. Call the reverse endpoint to reverse the direction of your sync operation ... mongosync replicates all writes that you performed on your new source cluster after you unblocked writes to the new destination cluster.
> 
> That way we would just have the cutover steps on the cutover tutorial. I think I'm in favor of this option now. (Apologies for the back and forth). @ltran-mdb2 @mvankeulen94 thoughts?

I think this setup makes sense and am also in favor of this for clarity + minimalism. @mvankeulen94 does this work for you?

Reply:
sgtm!